ui_framework/dist/ui_framework.css

```diff
@@ -557,6 +557,284 @@ table {
 .kuiCallOutHeader__title {
   color: #0079a5; }
 
+.kuiForm__textField {
```

Comment:
I ended up using a base class to style all the similar text fields (text, textarea, password, number...etc). I originally had this as a mixin, but figured having a base class saves us a lot of CSS.

You can see this class passed as a default class on top of every text based input... ex: `input type="password" className="kuiForm__textField kuiFormPassword"`.

I think that's the right way to do it.